### PR TITLE
Added types for algebra.js

### PR DESCRIPTION
**Pull request by CaselIT**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/algebra.js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  function parse(input: string): Equation | Expression;
+
+  class Equation {
+    lhs: Expression;
+    rhs: Expression;
+    toString(): string;
+  }
+
+  class Expression {
+    constants: Fraction[];
```
Comment (yuit) [inline types/algebra.js/index.d.ts]: should there be a constructor that take in string?

File: types/algebra.js/algebra.js-tests.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { Expression, Fraction, parse } from 'algebra.js';
+
+parse('x + 3');
+new Expression();
```
Comment (yuit) [inline types/algebra.js/algebra.js-tests.ts]: Could you add more test from the example in this page (http://algebra.js.org/)

File: types/algebra.js/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    terms: Term[];
+    simplify(): Expression;
+    constant(): Fraction;
+    toString(): string;
+  }
+
+  class Fraction {
```
Comment (yuit) [inline types/algebra.js/index.d.ts]: same here. Fraction class seems to take two number so should there be a constructor function to describe that

File: types/algebra.js/algebra.js-tests.ts (hunk 2 above)
Comment (CaselIT) [inline types/algebra.js/algebra.js-tests.ts]: I've updated the package

**Pull request merged**